### PR TITLE
Craft action scenario tests

### PR DESCRIPTION
**Pull request by U-lis**
Add tests for craft actions to prepare new feature: pet

Works form now makes sure all the actions are working properly before pet comes in.
After this PR is merged, craft actions with pet will be tested.

- [x] CombinationEquipment
- [x] CombinationConsumable
- [x] EventEquipmentItemCrafts
- [x] EventMaterialItemCrafts
- [x] RapidCombination

### CODE REVIEW

File: Lib9c/Model/EnumType/CraftType.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace Nekoyume.Model.EnumType
+{
+    public enum CraftType
+    {
+        Normal = 0,
+        Premium = 1,
+        Mimisbrunner = 2
```
Comment (Unengine) [inline Lib9c/Model/EnumType/CraftType.cs:7]:
```suggestion
        Mimisbrunnr = 2
```

File: .Lib9c.Tests/Action/Scenario/ItemCraftTest.cs [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+            Assert.Equal(recipeList.Count, inventoryState.Items.Count);
+            foreach (var itemId in targetItemIdList)
+            {
+                // TEST: Created equipment should match with targetItemList
+                Assert.NotNull(inventoryState.Items.Where(e => e.item.Id == itemId));
+            }
+        }
+
+        [Theory]
+        [InlineData(1, new[] { 200000 })] // 맛없는 요리
```
Comment (Unengine) [inline .Lib9c.Tests/Action/Scenario/ItemCraftTest.cs:150]: I think the test result of this inline data row is not helpful. Can you consider adding craftable recipe?

File: .Lib9c.Tests/Action/Scenario/ItemCraftTest.cs [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+                _avatarAddr
+            );
+
+            // Setup requirements
+            var random = new TestRandom(randomSeed);
+            var recipeList = _tableSheets.ConsumableItemRecipeSheet.OrderedList.Where(
+                recipe => targetItemIdList.Contains(recipe.ResultConsumableItemId)
+            ).ToList();
+            var allMaterialList = new List<EquipmentItemSubRecipeSheet.MaterialInfo>();
+            foreach (var recipe in recipeList)
```
Comment (Unengine) [inline .Lib9c.Tests/Action/Scenario/ItemCraftTest.cs:166]: I think count of the recipes should be checked. (Assert if count is non-zero or positive number)

File: .Lib9c.Tests/Action/Scenario/ItemCraftTest.cs (hunk 2 above)
Comment (U-lis) [reply .Lib9c.Tests/Action/Scenario/ItemCraftTest.cs:150]: > I think the test result of this inline data row is not helpful. Can you consider adding craftable recipe?

Oh, I just rechecked `ConsumableItemRecipeSheet` and 20`1`000 was what I originally attended.
Thanks for pointing out!

**Review by Unengine — APPROVED** [review]

**Pull request merged**